src/components/whiteframe/README.md

```diff
@@ -0,0 +1,23 @@
+# MdWhiteframe
+
+MdWhiteframe is a component that applies an elevation shadow to an element.
+
+## Screenshot
+<img src="http://www.klimtoren.be/content/material2/mdwhiteframe-example.png">
```

Comment:
The Material team itself host their images [there](https://github.com/angular/code.material.angularjs.org). So I think they will host the screenshot their self.